lib/git/changed-chunks.js

```diff
@@ -44,7 +44,7 @@ class ChangedChunks {
 	}
 }
 
-function compareFileNames(fileNameA, fileNameB) {
+function compareFileNames(fileNameA = '', fileNameB = '') {
```

Comment:
I think I'd prefer if we filtered out these bad filenames earlier or put guard code around calling `compareFileNames`, otherwise, this definition now suggests that all arguments are optional, which shouldn't be the case. I won't let this block the fix though